### ISSUE TITLE
Interference with HTML <sub> and <sup> tags

### ISSUE THREAD

**Issue by bhbr** [mathquill/mathquill#708]
Some websites change the size and offset of text subscripts and superscripts, e. g. so that the line height remains uniform. This messes with MQ's careful positioning of indices and exponents, roots and overlines. I have noticed this while integrating MQ into Squarespace.

<img width="255" alt="bildschirmfoto 2016-12-29 um 23 33 04" src="https://cloud.githubusercontent.com/assets/13440601/21556487/06143c04-ce21-11e6-8134-febac22d1960.png">

It would make it easier for people to integrate MQ into their website designs if this inheritance from plain HTML were cut at some level.

The offending CSS is (from Squarespace's normalize.css):

`/**`
`* Prevent 'sub' and 'sup' affecting 'line-height' in all browsers. */`
`*/`
`sub,`
`sup {`
`  font-size: 75%;`
`  line-height: 0;`
`  position: relative;`
`  vertical-align: baseline;`
`}`
`sup {`
`  top: -0.5em;`
`}`
`sub {`
 ` bottom: -0.25em;`
`}`

**Comment by bhbr** [comment]
That was probably not the correct passage, I couldn't backtrace it properly.
But in any case the problem should circumvent any such outside CSS.

**Comment by kschaefe** [comment]
Is there a reason to even use `sup` and `sub` instead of `span` tags with custom classes?

**Comment by bhbr** [comment]
Precisely my point. Why does MQ use `sup` and `sub`? Could they be renamed e. g. `mqsub` and `mqsup` so that they don't inherit outside CSS?

**Comment by laughinghan** [comment]
No, I don't believe there's any reason for to use `<sup>` and `<sub>` instead of plain `<span>`s. I originally did that in a misguided attempt to make the HTML be "semantic" because I hadn't yet realized the W3C was full of it. (I wanted the HTML for `x^2` to be `<var>x</var><sup>2</sup>`.) There don't appear to be any practical CSS-only solutions ([the `all` property](http://caniuse.com/css-all) is certainly not ready for prime time), so I think your suggestions are exactly right.

I think we may have been aware that this is potentially a problem since at least when we realized we needed to prefix all our CSS classes with `mq-`, but it's good to know about a concrete case (damn it, Squarespace). Mind submitting a PR?
